### DISCUSSION TITLE
Intersection Check-Box seems to be inverted

### DISCUSSION THREAD

**Discussion by Paul8043** [bernhard-42/vscode-ocp-cad-viewer#62] [Q&A]
Intersection check-box seems to have inverted functionality.

```
from build123d import *
from ocp_vscode import *

set_port(3939)

b = Box(10,20,30)

show_all()
```
Check-Box is checked:
![Intersection-checked](https://github.com/bernhard-42/vscode-ocp-cad-viewer/assets/127483918/af708a1d-2451-411c-8775-9739eb80a0b3)

Check-Box is un-checked:
![Intersection-unchecked](https://github.com/bernhard-42/vscode-ocp-cad-viewer/assets/127483918/016f0d7a-d834-47ae-9b7b-72ab133d090c)

**Comment by bernhard-42** [comment]
It actually works as designed:
![image](https://github.com/bernhard-42/vscode-ocp-cad-viewer/assets/954408/f55745bf-56b8-495b-8c5c-8724f3f3d929)

versus
![image](https://github.com/bernhard-42/vscode-ocp-cad-viewer/assets/954408/372fc729-620a-4a05-ab99-d74549bded56)
